java/src/main/java/org/rocksdb/AdvancedMutableColumnFamilyOptionsInterface.java

```diff
@@ -80,6 +80,25 @@ T setMemtablePrefixBloomSizeRatio(
    */
   double memtablePrefixBloomSizeRatio();
 
+  /**
+   * Enable whole key bloom filter in memtable. Note this will only take effect
+   * if memtable_prefix_bloom_size_ratio is not 0. Enabling whole key filtering
+   * can potentially reduce CPU usage for point-look-ups.
+   *
+   * Default: false (disable)
+   *
+   * @param memtableWholeKeyFiltering the memtable whole key filtering mode
+   * @return the reference to the current options.
+   */
+  T setMemtableWholeKeyFiltering(boolean memtableWholeKeyFiltering);
+
+  /**
+   * Returns whether a memtable whole key filtering mode will be used
```

Comment:
Same thing about comments "Returns whether whole key bloom filter is enabled in memtable"